modules/imgproc/src/hal_replacement.hpp

```diff
@@ -880,6 +880,25 @@ inline int hal_ni_gaussianBlur(const uchar* src_data, size_t src_step, uchar* ds
 #define cv_hal_gaussianBlur hal_ni_gaussianBlur
 //! @endcond
 
+/**
+   @brief Blurs an image using a symmetric Gaussian filter with square kernel and sigma=0.
+   @param src_data Source image data
+   @param src_step Source image step
+   @param dst_data Destination image data
+   @param dst_step Destination image step
+   @param width Source image width
+   @param height Source image height
+   @param depth Depth of source and destination image
+   @param cn Number of channels
+   @param ksize Width of kernel
+   @param border_type Border type
+*/
+inline int hal_ni_gaussianBlurSymmetric(const uchar* src_data, size_t src_step, uchar* dst_data, size_t dst_step, int width, int height, int depth, int cn, size_t ksize, int border_type) { return CV_HAL_ERROR_NOT_IMPLEMENTED; }
+
+//! @cond IGNORED
+#define cv_hal_gaussianBlurSymmetric hal_ni_gaussianBlurSymmetric
```

Comment:
also, why word "Symmetric" in the name? All Gaussian kernels are symmetric by definition

Reply:
Let it be Binomial